snapcraft/internal/pluginhandler.py

```diff
@@ -324,23 +324,9 @@ def migratable_fileset_for(self, step):
         return _migratable_filesets(fileset, self.code.installdir)
 
     def _organize(self):
-        organize_fileset = getattr(self.code.options, 'organize', {}) or {}
+        fileset = getattr(self.code.options, 'organize', {}) or {}
```

Comment:
This predates the days we had a jsonschema, so it *was* necessary, I am a little hesitant to remove it now though.